custodial-copy-db-builder/src/test/scala/uk/gov/nationalarchives/builder/MainSpec.scala

```diff
@@ -111,6 +113,7 @@ class MainSpec extends AnyFlatSpec with BeforeAndAfterEach with BeforeAndAfterAl
     firstFile.ingestDateTime.get should equal(Instant.parse("2024-07-03T11:39:15.372Z"))
     firstFile.sourceId.get should equal("SourceID")
     firstFile.citation.get should equal("Citation")
+    firstFile.consignmentRef.get should equal("TDR-2025-RNDM")
   }
 
```

Comment:
I can't put a comment on the line but there's a test `"runBuilder" should "return empty string for missing metadata values"`
Can you check that the consignmentRef is empty in the same way we are on those other fields please.